src/System.Net.Http.WinHttpHandler/tests/project.lock.json

```diff
@@ -173,10 +173,10 @@
           "lib/dotnet/System.IO.FileSystem.Primitives.dll": {}
         }
       },
-      "System.Linq/4.0.0-beta-22816": {
+      "System.Linq/4.0.0-beta-22703": {
         "dependencies": {
-          "System.Collections": "4.0.10-beta-22816",
-          "System.Runtime": "4.0.20-beta-22816"
+          "System.Collections": "4.0.0-beta-22703",
+          "System.Runtime": "4.0.0-beta-22703"
```

Comment:
Why are the numbers going backwards in some of these updates, i.e. from 22816 to 22703?  That doesn't seem right.  This comment applies generally to the entire file.